Files/Interacts/Interaction.cs

```diff
@@ -315,6 +315,13 @@ public void OpenItemWithApplicationPicker_Click(object sender, RoutedEventArgs e
             OpenSelectedItems(true);
         }
 
+        public void OpenFileLocation_Click(object sender, RoutedEventArgs e)
+        {
+            var item = CurrentInstance.ContentPage.SelectedItem as ShortcutItem;
+            var folderPath = Path.GetDirectoryName(item.TargetPath);
```

Comment:
Shortut `TargetPath` can somatimes be null or empty. Please check if this causes issues.
The most notable example of this is the "Vault" link in the OneDrive folder

Reply:
I could only get the TargetPath empty/null by manually setting it to empty/null during debugging.
But when it is null/empty nothing happens. Do you have any suggestions on how we should handle this?